libraries/api/discussion_helper.cpp

```diff
@@ -191,7 +279,9 @@ namespace golos { namespace api {
 //
 // set_url
     void discussion_helper::impl::set_url(discussion& d) const {
-        const comment_api_object root(database().get<comment_object, by_id>(d.root_comment), database());
+        comment_object cm = database().get<comment_object, by_id>(d.root_comment);
+
+        comment_api_object root = create_comment_api_object( cm );
```

Comment:
It is to heavy to create additional comment_api_object.
Refactor this to get values comment and comment_content.

Reply:
Well, then I should refactor callback in `social_network` plugin to make it return just content and nothing more.